scripts/ping_issue_assignees.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime as dt
+import os
+
+from github import Github
+
+
+def main():
+    g = Github(os.environ["GITHUB_TOKEN"])
+    repo = g.get_repo("allenai/allennlp")
+    open_issues = repo.get_issues(state="open")
+
+    for issue in open_issues:
+        if (
+            issue.milestone is None
```

Comment:
Do we need this check, and the PR check?  I'd be fine if it just said "is assigned and last updated more than two weeks ago".